pkg/parse/parse.go

```diff
@@ -177,12 +168,7 @@ func getFlags(ldApi ld.ApiClient) ([]string, error) {
 }
 
 func (b *branch) findReferences(cmd git.Git, flags []string, ctxLines int, exclude *regexp.Regexp) (grepResultLines, error) {
-	err := cmd.Checkout()
```

Comment:
sanity check - do github/bitbucket give us the repo with the right branch already checked out?

Reply:
Yes, I checked bb and github, and they appear to give us the correct branch checkout. I'll make a release candidate out of this branch and double check as well.